nodes/clarify_insert.js

```diff
@@ -148,6 +148,12 @@ module.exports = function (RED) {
     this.on('input', async function (msg, send, done) {
       // Validate incoming id. Must be correct to continue
       let id = RED.util.getMessageProperty(msg, config.signalId);
+      if (id === undefined) {
+        let errMsg = 'missing signal id';
+        node.status({fill: 'red', shape: 'ring', text: errMsg});
+        done(errMsg);
+        return;
+      }
       if (!signalIdPattern.test(id)) {
```

Comment:
Didn't this pickup `id === undefined`?

Reply:
nope, it did not. :/

Reply:
Weird. Anyways. Nice to get an explicit error message when the id isn't defined.